tests/stdlib/toldparsehex.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  matrix: "--d:oldParseHex"
```

Comment:
nimLegacyParseInts ? (since we use `nimLegacy` convention already, and since it's more than parseHex)

Reply:
> nimLegacyParseInts 
parseInts is not very clear?